models/issue_label.go

```diff
@@ -127,6 +127,33 @@ func (label *Label) ForegroundColor() template.CSS {
 	return template.CSS("#000")
 }
 
+func initalizeLabels(e Engine, repoID int64, labelTemplate string) error {
```

Comment:
Could you reuse this function here: https://github.com/go-gitea/gitea/blob/f9ec2f89f2265bc1371a6c62359de9816534fa6b/routers/repo/issue_label.go#L36

Reply:
I actually did the opposite. The router now uses _this_ function.

Reply:
Oh, sorry did not notice you have already done this :+1: